src/main/java/net/mcreator/element/types/VillagerTrade.java

```diff
@@ -63,4 +63,18 @@ public static class Entry {
 			public double priceMultiplier;
 		}
 	}
+
+	public boolean hasWanderingTrade() {
```

Comment:
This should be checked inside `WorkspaceInfo`, it could be a method like:
```java
	public boolean hasVillagerTrades(boolean wandering) { ... }
```

Reply:
Yet, method `hasCustomRenderer()` for PotionEffect mod element doesn't have a method check...